docs/runbooks/HCOGoldenImageWithNoSupportedArchitecture.md

```diff
@@ -0,0 +1,207 @@
+
+For pre-defined DICTs, this annotation is already set, but for custom DICTs
+(user defined DICTs), this annotation must be set by the user in the
+HyperConverged custom resource (CR).
+
+For each DICT, if the annotation does not include any architecture that is
+supported by the cluster (which mean, there is no node in the cluster that with
+any of the architectures listed in the DICT annotation), Then HCO will trigger
+the `HCOGoldenImageWithNoSupportedArchitecture` alert for this specific DICT.
+
+## Impact
+
+When this alert is triggered, it means that the DICT is not supported by any of
+the nodes in the cluster. HCO will not populate the SSP CR with this DICT, and
+so this golden image will not be available for use in the cluster.
+
+## Diagnosis
+
+Read the HyperConverged CR:
+
+```bash
+   # Get the namespace of the HyperConverged CR
+$ NAMESPACE="$(kubectl get hyperconverged -A --no-headers | awk '{print $1}')"
+   
+#Read the HyperConverged CR
+$ kubectl get hyperconverged -n "${NAMESPACE}" -o yaml
+```
+
+There are a few fields in the HyperConverged CR status that can be used to
+diagnose this issue:
+
+1. The `status.nodeInfo.workloadsArchitectures` shows the list of architectures
+   supported by the cluster.
+2. The `status.dataImportCronTemplates` field shows the list of DICTs that are
+   defined in the cluster.
+    1. For each DICT, in this list, check the
+       `ssp.kubevirt.io/dict.architectures` annotation. Unlike the annotation
+       in the spec field, this annotation contain only the architectures that
+       are supported by the image **and** byt the cluster.
+
+       If the annotation is empty, then there is no architecture supported by
+       the image and by the cluster.
+    2. In this case, the status field of this DICT will include the
+       `conditions` field, with the `Deployed` condition set to `False`, and
+       the `reason` field set to `UnsupportedArchitectures`.
+    3. The DICT's `status.workloadsArchitectures` field shows the list of
+       architectures supported by the image, as was set in the
+       `ssp.kubevirt.io/dict.architectures` annotation in the source DICT.
+
+### Example
+
+```yaml
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+...
+status:
+  ...
+  dataImportCronTemplates:
+      - metadata:
+        annotations:
+          ssp.kubevirt.io/dict.architectures: ""
+        name: my-image
+        spec:
+          ...
+        status:
+          conditions:
+            - message: DataImportCronTemplate has no supported architectures for the current
+                cluster
+              reason: UnsupportedArchitectures
+              status: "False"
+              type: Deployed
+          originalSupportedArchitectures: someUnsupportedArch,otherUnsupportedArch
+```
+
+## Mitigation
+
+### Pre-defined DataImportCronTemplates
+
+These DICTs are not defined in the HyperConverged CR, but are defined in the
```

Comment:
What does this mean. What is the difference between HyperConverged CR and HCO itself?
Please treat the users as though they are not familiar with this differences